### ISSUE TITLE
Add "Ignore All" button during Packs update

### ISSUE THREAD

**Issue by bureaua** [gnu-mcu-eclipse/eclipse-plugins#285]
### Description
A lot of messages with "Illegal PDSC file size" during Packs update
![2018-03-31 16-50-10](https://user-images.githubusercontent.com/739921/38163969-87d292ee-3505-11e8-8a36-2e8723407815.png)

### Steps to Reproduce
- Download and run eclipse-oxygen-3-x86_64
- Open C/C++ Packs (Show View)
- Press update button on Packs tab

## Versions
[plug-in version]
4.3.1.201801301501

### [Eclipse version]
Oxygen.3 Release (4.7.2)

### [Java version]
java 9.0.4

### [operating system]
Ubuntu 16.04 x64
Windows 10 x64

**Comment by ilg-ul** [comment]
yes, unfortunately sometimes the servers do not respond and the plugin reports errors.

**Comment by bureaua** [comment]
Will it possible to ignore this message in automatic way? Because I have to press now "Ignore" button each time, many times to continue.

**Comment by ilg-ul** [comment]
If the plugin will automatically ignore them, you'll think that everything went well, and you'll be surprised when those packs have no content, or were not updated.

just curious, during the lengthy update process, did you check the messages in the Packs Console?

**Comment by bureaua** [comment]
> in the Packs Console?
```
Missing "/opt/eclipse-oxygen-3-x86_64/Packages/.cache/NXP.QN908x_DFP.1.1.4.pdsc", ignored by user request.
Processing "/opt/eclipse-oxygen-3-x86_64/Packages/.cache/Keil.MCB1500_BSP.1.2.0.pdsc" for content.xml ...
Schema version "1.2".
```
Should I expect something interesting?


> you'll think that everything went well

Agree. May be Check Button for this, like "ignore this message in future"...

**Comment by ilg-ul** [comment]
I added an 'Ignore All' button.

**Comment by ilg-ul** [comment]
Fixed on 2018-04-03.

**Comment by bureaua** [comment]
By using which link I can try to test ?
http://gnu-mcu-eclipse.netlify.com/v4-neon-updates-test -- does not see any updates

**Comment by ilg-ul** [comment]
The 4.3.3.201804051758 pre-release is now available for testing from http://gnu-mcu-eclipse.netlify.com/v4-neon-updates-test.

**Comment by ilg-ul** [comment]
Fixed since 4.3.3.201804191501.
